flocker/node/_docker.py

```diff
@@ -564,11 +564,15 @@ def _list():
                 image_data = self._client.inspect_image(image)
                 unit_environment = []
                 container_environment = data[u"Config"][u"Env"]
```

Comment:
When I run `docker inspect busybox:latest` I get a `Config` and `ContainerConfig` keys...but perhaps that's a peculiarity of `docker inspect` which can inspect both images and containers.